carservice/src/main/java/com/senlainc/warsaw/tyurin/controller/OrderController.java

```diff
@@ -0,0 +1,108 @@
+package com.senlainc.warsaw.tyurin.controller;
+
+import com.senlainc.warsaw.tyurin.dto.OrderDto;
+import com.senlainc.warsaw.tyurin.entity.Order;
+import com.senlainc.warsaw.tyurin.exception.CriteriaNotFoundException;
+import com.senlainc.warsaw.tyurin.service.IOrderService;
+import com.senlainc.warsaw.tyurin.util.builder.OrderBuilder;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
+@RequestMapping("orders")
+public class OrderController {
+
+    private final OrderBuilder orderBuilder;
+    private IOrderService orderService;
+
+    public OrderController(OrderBuilder orderBuilder,
+                           IOrderService orderService) {
+        this.orderService = orderService;
+        this.orderBuilder = orderBuilder;
+    }
+
+    @ResponseBody
+    @PostMapping()
+    public void create(@Valid @RequestBody OrderDto orderDto) {
```

Comment:
Void is totally fine for restfull apis, because it reduce code complication, and can better describe real events happened in server

Reply:
But be aware when using void you are loosing and opportunity to send additional info except http code, and sometimes it can be challenging to handle exceptions

Reply:
Usually even if it's add or delete operation i'm using returning type to be able create more describable responses. But it's totally fine to have it like now. Just be aware